packages/manager/modules/ng-layout-helpers/package.json

```
@@ -29,7 +29,11 @@
   },
   "peerDependencies": {
     "@ovh-ux/ng-ovh-api-wrappers": "^4.0.6",
+    "@ovh-ux/ng-translate-async-loader": "^2.1.0",
     "@ovh-ux/ui-kit": "^4.2.0",
-    "angular": "^1.7.9"
+    "@uirouter/angularjs": "^1.0.22",
+    "angular": "^1.7.9",
+    "angular-translate": "^2.18.1",
+    "bootstrap": "^4.0.0"
```

```suggestion
    "bootstrap4": "^4.0.0"
```
should be specified as in some apps we use `bootstrap: ^3.3.7`

---

This is done :)